### DISCUSSION TITLE
What should be in a service footer?

### DISCUSSION THREAD

**Discussion by tomyems** [UKHomeOffice/design-system#349] [Ideas]
The standard Home Office Design System footer looks like:
![image](https://user-images.githubusercontent.com/93599300/157253240-39f25de3-65b9-4d7b-a8dc-0f002de9a508.png)
(Source: https://design.homeoffice.gov.uk/components?name=Footer)

The GOV.UK design system suggests that footers should include:
- your department’s page on GOV.UK
- other language options
- terms and conditions
- privacy policy

There's also guidance in other design systems:
- https://hmcts-design-system.herokuapp.com/components/footer

For internal services, which is where this footer would be used otherwise it'd be the GOV.UK footer, I wonder what would be best. Accessibility statements and privacy policies spring to mind?

**Comment by mashukreza** [comment]
Gov.uk on the footer should be GOV.UK as per GDS style guide
